@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -628,6 +628,22 @@ export class MapControls extends EventDispatcher {
             const tiltFinished = this.m_tiltAnimationTime > this.tiltToggleDuration;
             if (tiltFinished) {
                 if (this.m_needsRenderLastFrame) {
+                    // There may be some numerical inaccuracies in the application of the
+                    // `deltaAngle` during the animation, hence we check one last time to see if we
+                    // need to fix the tilt
+                    const mapViewTiltRad = THREE.MathUtils.degToRad(this.mapView.tilt);
+                    const deltaAngle = (this.m_targetedTilt ?? mapViewTiltRad) - mapViewTiltRad;
+                    if (deltaAngle !== 0) {
+                        MapViewUtils.orbitAroundScreenPoint(
+                            this.mapView,
+                            0,
+                            0,
+                            0,
+                            deltaAngle,
+                            this.m_maxTiltAngle
+                        );
+                        this.updateMapView();
+                    }
```

Comment:
I think the call to this.stopTilt() is done to soon. It sets:
 this.m_targetedTilt = this.m_currentTilt = undefined;

Then later we pass to easeOutCubic() as target value this.targetedTilt, which returns the current mapView.tilt if this.m_targetedTilt  is undefined.

If the stopTilt() call is done after easOutCubic, together with your epsilon changes, it works fine for me.